jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2911,7 +2912,7 @@ code_blockt java_bytecode_convert_methodt::convert_store(
     bytecode_write_typet::VARIABLE,
     var_name);
 
-  block.add(code_assignt{var, toassign}, location);
+  block.add(code_assignt{var, op[0]}, location);
```

Comment:
:question: I'd expect to see a unit test / change to existing unit test for this change no?